.travis.yml

```diff
@@ -319,6 +320,31 @@ jobs:
       before_script:
         - bash ./tests/travis/install-postgres-10.sh
 
+    - stage: Test
+      php: 7.1
+      env: DB=pgsql POSTGRESQL_VERSION=11.0
+      sudo: required
+      services:
+        - docker
+      before_script:
+        - bash ./tests/travis/install-postgres-11.sh
+    - stage: Test
```

Comment:
Could you visually separate jobs by adding an empty line between them?

Reply:
Stages are currently separated by vendor&version so this is consistent with the rest.